components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/org.wso2.carbon.event.output.adapter.ui.endpoint/src/main/java/SuperTenantSubscriptionEndpoint.java

```diff
@@ -54,7 +53,7 @@ public void onOpen (Session session, @PathParam("streamname") String streamName
             log.debug("WebSocket opened, for Session id: "+session.getId()+", for the Stream:"+streamName);
         }
         PrivilegedCarbonContext carbonContext = PrivilegedCarbonContext.getThreadLocalCarbonContext();
-        carbonContext.setTenantId(-1234);
+        carbonContext.setTenantId(carbonContext.getTenantId());
         tenantId = carbonContext.getTenantId();
```

Comment:
tenant Id I have to change. I understand you can getting it from tenant flow and  No need to pass it. I kept it for now. I will make the change once I review it with you or suho on Monday